spec/cli/questionsAndAnswers.test.js

```diff
@@ -1,19 +1,44 @@
-import QuestionsAndAnswers from '../../client/src/components/QuestionsAndAnswers';
 import '@testing-library/jest-dom/extend-expect';
+import QuestionsAndAnswers from '../../client/src/components/QuestionsAndAnswers';
+import QandAForm from '../../client/src/components/QandAForm';
+import QandAList from '../../client/src/components/QandAList';
+import QandAItem from '../../client/src/components/QandAItem';
+import QandAButtons from '../../client/src/components/QandAButtons';
+import exampleData from '../../helpers/QandAExample.js';
```

Comment:
As someone about to split their widget into a lot of subcomponents, it's a relief to see that having so many doesn't seem to negatively impact the Lighthouse load times😅